Signal/src/network/PushManager.m

```diff
@@ -147,16 +148,16 @@ - (void)application:(UIApplication *)application
 
         if (threadId) {
             TSThread *thread = [TSThread fetchObjectWithUniqueID:threadId];
-            TSOutgoingMessage *message =
-                [[TSOutgoingMessage alloc] initWithTimestamp:[NSDate ows_millisecondTimeStamp]
-                                                    inThread:thread
-                                                 messageBody:responseInfo[UIUserNotificationActionResponseTypedTextKey]];
-            [self.messageSender sendMessage:message
+            NSString *replyText = responseInfo[UIUserNotificationActionResponseTypedTextKey];
+
+            [ThreadUtil sendMessageWithText:replyText
+                inThread:thread
+                messageSender:self.messageSender
```

Comment:
Leaving/adding people to a group does not affect disappearing time on Desktop/iOS, furthermore we intentionally don't automatically delete those status messages, so this seems sufficient, if not a bit tenuous.